.github/CODEOWNERS

```diff
@@ -23,17 +23,32 @@
 * @Qiskit/terra-core
 
 # Qiskit folders (also their corresponding tests)
-algorithms/           @Qiskit/terra-core @woodsp-ibm @ElePT
-opflow/               @Qiskit/terra-core @woodsp-ibm @ikkoham
-qiskit/utils/         @Qiskit/terra-core @woodsp-ibm
-providers/            @Qiskit/terra-core @jyu00
-quantum_info/         @Qiskit/terra-core @ikkoham
-qpy/                  @Qiskit/terra-core
-pulse/                @Qiskit/terra-core @eggerdj @wshanks
-synthesis/            @Qiskit/terra-core @alexanderivrii @ShellyGarion
-scheduler/            @Qiskit/terra-core @eggerdj @wshanks
-visualization/        @Qiskit/terra-core @nonhermitian
-primitives/           @Qiskit/terra-core @ikkoham @t-imamichi
+.azure/                @Qiskit/terra-core @Eric-Arellano
+.github/               @Qiskit/terra-core @Eric-Arellano
```

Comment:
These weren't in the bot; completely new rules. Whereas the rest below were all migrated